docs/input-manifest/providers/gcp.md

```diff
@@ -11,7 +11,7 @@ metadata:
-  gcpproject: cHJvamVjdC1jbGF1ZGll
+  gcpproject: cHJvamVjdC1jbGF1ZGll # base64 created from GCP project ID
```

Comment:
Can put this information somewhere where its more visible? i.e. [here](https://docs.claudie.io/v0.4.1/input-manifest/providers/gcp/)? 
```
GCP provider requires you to input multiline credentials as well as specific GCP project ID gcpproject where to provision resources.
```